src/components/ErrorsField.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import cn from 'classnames';
+import { Errors } from '../utils/Errors';
+
+interface Props {
+  showErrorMessage: boolean;
+  setShowErrorMessage: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
will be the same. Use this syntax in all such cases
```suggestion
  setShowErrorMessage: (isError: boolean) => void;
```